scene/main/instance_placeholder.cpp

```diff
@@ -75,6 +75,7 @@ String InstancePlaceholder::get_instance_path() const {
 	return path;
 }
 
+
```

Comment:
This introduces a style issue: https://travis-ci.org/godotengine/godot/jobs/641479127?utm_medium=notification&utm_source=github_status